manifests/dispatch_router.pp

```diff
@@ -0,0 +1,74 @@
+# == Class: capsule::dispatch_router
+#
+# Install and configure Qpid Dispatch Router
+#
+class capsule::dispatch_router (
+) {
+
+  class { 'qpid::router': }
+
+  # SSL Certificate Configuration
+  class { 'certs::qpid_router':
+    require => Package[$qpid::router::router_packages],
```

Comment:
Couldn't we just require `Class['qpid::config']` ? Should that class itself not be requiring this package to be present?